packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -825,14 +834,20 @@ class SliverReorderableListState extends State<SliverReorderableList> with Ticke
   @override
   Widget build(BuildContext context) {
     assert(debugCheckHasOverlay(context));
+    final SliverChildBuilderDelegate childrenDelegate = SliverChildBuilderDelegate(
+      _itemBuilder,
+      childCount: widget.itemCount + (_dragInfo != null ? 1 : 0),
+    );
+    if(widget.itemExtent != null)
+      return SliverFixedExtentList(
+        itemExtent: widget.itemExtent!,
+        delegate: childrenDelegate,
+      );
     return SliverList(
       // When dragging, the dragged item is still in the list but has been replaced
       // by a zero height SizedBox, so that the gap can move around. To make the
       // list extent stable we add a dummy entry to the end.
-      delegate: SliverChildBuilderDelegate(
-        _itemBuilder,
-        childCount: widget.itemCount + (_dragInfo != null ? 1 : 0),
-      ),
+      delegate: childrenDelegate,
```

Comment:
In addition I think the return value can be simplified a bit. This whole block might work better as:

```dart
    final SliverChildBuilderDelegate childrenDelegate = SliverChildBuilderDelegate(
      _itemBuilder,
      // When dragging, the dragged item is still in the list but has been replaced
      // by a zero height SizedBox, so that the gap can move around. To make the
      // list extent stable we add a dummy entry to the end.
      childCount: widget.itemCount + (_dragInfo != null ? 1 : 0),
    );

    return widget.itemExtent != null
      ? SliverFixedExtentList(
          itemExtent: widget.itemExtent!,
          delegate: childrenDelegate,
        )
      : SliverList(delegate: childrenDelegate);
```